src/olympia/abuse/models.py

```diff
@@ -379,7 +390,9 @@ class AbuseReport(ModelBase):
     collection = models.ForeignKey(
         Collection, null=True, related_name='abuse_reports', on_delete=models.SET_NULL
     )
-    message = models.TextField(blank=True)
+    message = models.TextField(
+        blank=True, help_text='The body/content of the abuse report (required).'
```

Comment:
It's not required if a reason is sent

Reply:
I copy pasted from the existing docs. So I guess those docs are incorrect?

[<img width="522" alt="image" src="https://github.com/mozilla/addons-server/assets/19595165/0b547cb2-a156-431b-ae38-a3512f1faea2">](https://addons-server.readthedocs.io/en/latest/topics/api/abuse.html#post--api-v5-abuse-report-addon-)